### PR TITLE
Only download toplevel outputs

### PR DESCRIPTION
**Pull request by fejta**
Also have bazelisk use 0.28.1 for this repo

/assign @Katharine @cjwagner 
/hold

https://docs.bazel.build/versions/master/command-line-reference.html#build-options

### CODE REVIEW

File: .bazelrc [hunk 1]
```diff
@@ -45,8 +45,20 @@ build:remote --define=EXECUTOR=remote
 build:remote --remote_executor=grpcs://remotebuildexecution.googleapis.com
 build:remote --remote_timeout=3600
 
+# Minimize what is downloaded
+build:inmemory --experimental_inmemory_jdeps_files
+build:inmemory --experimental_inmemory_dotd_files
+
+# Minimize what is downloaded
+build:toplevel --config=inmemory
+build:toplevel --experimental_remote_download_outputs=toplevel
```
Comment (clarketm) [inline .bazelrc]: Could potentiallt reduce some boilerplate by using ([ref](https://docs.bazel.build/versions/master/command-line-reference.html#flag--experimental_remote_download_toplevel))
```
build:toplevel --experimental_remote_download_toplevel
build:minimal --experimental_remote_download_minimal
```

Comment (fejta) [reply .bazelrc]: Try it out and fail spectacularly.

Comment (clarketm) [reply .bazelrc]: You are right, looks like those flags are on `master` docs but not in `0.28.1`.

**Comment by fejta** [comment]
```console
ERROR: Failed to run workspace status command hack/print-workspace-status.sh: /home/fejta/.cache/bazel/_bazel_fejta/c229eee16670a3bb15d65c08362dcf5d/execroot/io_k8s_test_infra/bazel-out/stable-status.txt (Permission denied)
```

If you see this error it looks like you can resolve it with a `bazel clean`

**Pull request merged**